### ISSUE TITLE
Google Pixel 8, carstream showing but, no new messages on this drive, also no Fermata auto, it does not show up on android auto. Any help highly appreciated.

### ISSUE THREAD

**Issue by Abey0712** [shmykelsa/AAAD#269]

**Comment by yukiyamaiina** [comment]
It's the same phenomenon ><

**Comment by biggerald1** [comment]
Turn on developer mode in android auto under settings>connected devices>connection preferences>android auto, then enable developer mode & allow unknown sources under android auto's developer settings. That will get apps like carstream showing on ur vehicle infotainment screen.

HOWEVER, I don't know anybody getting it to work yet on any Pixel w/ Android 14 unless they had it working prior to installing 14. For some weird "grandfathered" reason AAAD, carstream, & other AAAD apps will work if u had them working & were using them PRIOR to updating to 14. If ur on 14 & are just now downloading AAAD or any of its apps it's too late. U have to downgrade to 13 & get AAAD & the apps u want THEN upgrade to 14 AFTER for it to work on Pixels. Not sure if that's even possible on a P8 since it shipped w/ Android 14. It works on Samsungs no problem.

It's weird cause I have a Pixel Fold on 13. I had AAAD/carstream working on 13 for 2 weeks so I bought AAAD Pro. I updated tht phone to 14 & it still worked. Got a replacement Pixel Fold for an issue & I updated tht phone to 14 BEFORE installing AAAD/carstream & now I get the dreaded "no new messages during this drive" & no AAAD apps work.

This behavior has been verified by many ppl. If u install AAAD & car stream & use them BEFORE updating to Android 14 ur fine. If ur on 14 first & trying to install/use them they don't work. U can get em to show in ur car but you'll get the no new msgs error. There's at least 4 ppl tht downgraded to 13 & they worked again. I can't downgrade because I can't unlock my bootloader. 

Long story short ur screwed unless u can/want to downgrade ur new Pixel to 13. The developer hasn't responded to any msgs so idk if he is looking into it or is bothered to care. I bought pro but haven't been helped or responded to yet. I'd downgrade to 13 if I could because carstream is worth it to me. I miss it dearly.

**Comment by mdeguzis** [comment]
Is there a fork that is trying to make things up to date with Android 14?

**Comment by biggerald1** [comment]
Not to my knowledge. They're still selling the pro version w/o mentioning the a14 issue. I've been trying to find or create a fix myself. I have made progress where I can force the removal of package installer v14 & force the install of package installer v13 on android 14. This fixes it however once you reboot or loose cell connection or switch to Wi-Fi it causing the phone to crash & soft brick where u have to load the os & lose all data. It has something to do w/ package installer v14 being baked into the android 14 os. Im assuming the phone looks for something tht isn't there or in a proper place causing the crash.

**Comment by ranasrule** [comment]
same issue here on Android 14 on Pixel 7 Pro and latest version of AAAD (with pro subscription)

**Comment by christopherquenneville** [comment]
Android 14 not working - has anyway found an APK like the one that was working in Android 13?

**Comment by Ken-Crowley** [comment]
Same issue 😭

**Comment by muppetofdoom** [comment]
Is there any news on how close a fix is? 😁

**Comment by biggerald1** [comment]
The developer is selling the app w/o mentioning it doesn't work anywhere. There has been no acknowledgement of it being a problem on here or anywhere. They're not replying to anything & continuing as if it doesn't exist. Im actually not even sure they check GitHub. According to 1 person on here they are aware as he said he was refunded through DM's for it not working. If they're working on a fix they're doing so in full stealth mode. It doesn't seem like too much to ask to do 1 of the following:
A. Say "We aren't actually trying to fix this so contact us for a refund if purchased after this date ##/##/####.
B. "We're working on a fix but it may be some time. I will post regular updates on progress "
C. If you don't plan to fix it or even try or even have the human decency to say anything AT LEAST post a banner on the app before somebody pays saying it won't work on a14 or pixels for people. 
It just seems SUPER foul to just sell this knowing it's in a worthless state right now.

**Comment by mf-stevens** [comment]
While it does seem that the issue queue is being ignored, the "Help" option in the app itself got me a reply right away. I made sure to keep it extremely civil, because although anger is appropriate after paying for something that doesn't work, I wanted to maximize chances of a reply. Turns out that works SUPER WELL, people -- response time was less than 8 hours!!

> we are indeed aware of problems affecting Android 14 and we are actively working on it. It's mostly a cat chasing the mouse game. Google likes taking measures to stop boredom in their system, but we are working on the fix. There's a lot we are testing so I can't provide an ETA right now, I'm afraid.

I then asked if I could share that with the issue queue:

> Feel free to do so!
> GR

**Comment by yeahforbes** [comment]
While it does seem that the issue queue is being ignored, the "Help" option in the app itself got me a reply right away. I made sure to keep it extremely civil, because although anger is appropriate after paying for something that doesn't work, I wanted to maximize chances of a reply. Turns out that works SUPER WELL, people -- response time was less than 8 hours!!

> we are indeed aware of problems affecting Android 14 and we are actively working on it. It's mostly a cat chasing the mouse game. Google likes taking measures to stop boredom in their system, but we are working on the fix. There's a lot we are testing so I can't provide an ETA right now, I'm afraid.

I then asked if I could share that with the issue queue:

> Feel free to do so!
> GR

**Comment by biggerald1** [comment]
> While it does seem that the issue queue is being ignored, the "Help" option in the app itself got me a reply right away. I made sure to keep it extremely civil, because although anger is appropriate after paying for something that doesn't work, I wanted to maximize chances of a reply. Turns out that works SUPER WELL, people -- response time was less than 8 hours!!
> 
> > we are indeed aware of problems affecting Android 14 and we are actively working on it. It's mostly a cat chasing the mouse game. Google likes taking measures to stop boredom in their system, but we are working on the fix. There's a lot we are testing so I can't provide an ETA right now, I'm afraid.
> 
> I then asked if I could share that with the issue queue:
> 
> > Feel free to do so!
> > GR
I appreciate you. My replies got more & more negative as a lot of time passed w/o any news or acknowledgement. That coupled w/ the fact you can buy this now w/o being told or warned it doesn't work. Expecting a reply in a GitHub where you get the app isn't toooo crazy of an expectation. Glad to see at least some effort though. Ty for taking the time to post for the developers.

**Comment by muppetofdoom** [comment]
Hi, and thanks for posting, good to hear work is ongoing in the background, my charging sessions just aren't the same at the moment 😕 🤣

**Comment by ahinks** [comment]
i bought this app not knowing its not working on pixel devices and really hope this functionality gets restored as i have not been able to actually use the app sense purchase.

**Comment by danielweipinglee** [comment]
I miss my working CarStream very much 😞

**Comment by akalongman** [comment]
Still no ETA?

**Comment by lmntngrn** [comment]
I reached out to the developer. I received the same reply as yeahforbes. Verbatim. I am not holding my breath for the time being. I anticipate at least few months if ever.

**Comment by muppetofdoom** [comment]
Hi all , i reached out via the help option and have recieved the following reply

_

> "Yes progress were made indeed. At the moment we are trying to bundle the solution inside AAAD itself because as of now we found a method that requires much configuration from the start. "

_

**Comment by ben125125** [comment]
This is promising @muppetofdoom keep me updated i am considering rolling back as I just installed a head unit in my old car and went to get AAAD and realized all of the apps are bricked on android 14 with my pixel 6

**Comment by GTO-MAN** [comment]
After paying for the AAAD app, I initialized the smartphone, reinstalled it, and ran it again, but it prompted me to pay again... Either refund 3.5 euros or give me pro rights again... or give me the license again... If not, I'll refund the money to the payment card company. I will demand it..!!

**Comment by lmntngrn** [comment]
> After paying for the AAAD app, I initialized the smartphone, reinstalled it, and ran it again, but it prompted me to pay again... Either refund 3.5 euros or give me pro rights again... or give me the license again... If not, I'll refund the money to the payment card company. I will demand it..!!

Take it easy. Just use the help function on the app and explain your situation. They are very responsive and will reinstate your pro license so long as you can prove that you've paid for it. You won't get too far with that demeanour.

**Comment by GTO-MAN** [comment]
We will send you proof of payment..!! I still can't change to AAAD App Pro...!! Please solve it..!! Otherwise, please send me a re-download link (pro account available)..!!


-----Original Message-----

**Comment by thespyman13** [comment]
> Hi all , i reached out via the help option and have recieved the following reply
> 
> _
> 
> > "Yes progress were made indeed. At the moment we are trying to bundle the solution inside AAAD itself because as of now we found a method that requires much configuration from the start. "
> 
> _

Any update?

**Comment by danielweipinglee** [comment]
Any update?

**Comment by Molorero** [comment]
Any at all????

**Comment by mdeguzis** [comment]
Please stop just asking for updates when there clearly are not any. The dev is working on it if you have read above. Send a support request from the app via email if you're curious.

**Comment by ko0ky313** [comment]
> Since I have AAAD a bit longer for now but never really used it, only Fermata, I now wanted to set up Screen2Auto. Obviously didn't work (Pixel 5, Android 14). My search brought me here. Nice to hear that updates are in development even if it takes time.
> 
> My question is: Why is it that Pixels don't work? Is it in general or only Pixel that already come with Android 14?
> 
> Is it possible to roll back my Pixel 5 to Android 13? How difficult is it? Any Guideline?
> 
> Just want to make sure if it's worth the try.
> 
> Thx guys

Maybe its android 14?? I'm using a samsung device and the app I'm using stopped working (carstream).

> [GTO-MAN](https://github.com/GTO-MAN) commented [on Dec 21, 2023]
> After paying for the AAAD app, I initialized the smartphone, reinstalled it, and ran it again, but it prompted me to pay again... Either refund 3.5 euros or give me pro rights again... or give me the license again... If not, I'll refund the money to the payment card company. I will demand it..!!

OMG this guy.. acting this way for 3.50 euro, he comes across as the devs owe him their lives and 100% attention.

**Comment by lmntngrn** [comment]
Reached out to the developer. Asked him/her if there is a timeline for a fix. Here's the reply:

Not as of now. There’s a test version we’ve distributed to a closed group. Though we're finding out some errors we’re still managing to get It fixed.

**Comment by GipsyChimp** [comment]
The question is: which Version of AA und which version of Screen2Auto where the last, that worked on Android 13. I rolled back already and want to get it to Work. But so far not successful

**Comment by FurkanVG** [comment]
Is it possible to join the closed group to test? Id love to test it.

**Comment by guesswhomf** [comment]
is everyone still coming here for updates?

**Comment by mdeguzis** [comment]
> > is everyone still coming here for updates?
> 
> Of course :)

And annoyingly filling my inbox with spam instead of just watching the thread

**Comment by kaheil** [comment]
Can I get the test version. Samsung Galaxy s23 Android 14

**Comment by ahinks** [comment]
If you need another im on a pixel 7 pro running android 14, i bought it when this problem started (didn't know there was a issue with compatibility) didn't want to return it as i would like to see it work. If i can help in anyway pm me.

**Comment by JCoyk** [comment]
Any updates? So I just got a s24 ultra and it automatically has 15 installed on it. I was wondering if just downgrading to 13 and then installing it back on here is the only solution as of now? This is too bad! I have also paid for the Pro version 🫤

**Comment by guesswhomf** [comment]
I see that Fermata Auto APK (V1.9.9 Updated) was updated on 3/18/24 [](https://aaadapk.com/fermata-auto-apk/) has anyone gotten it to work with Android 14?

**Comment by danielweipinglee** [comment]
> I see that Fermata Auto APK (V1.9.9 Updated) was updated on 3/18/24 [](https://aaadapk.com/fermata-auto-apk/) has anyone gotten it to work with Android 14?

No luck here 😕

**Comment by miticanicu** [comment]
Today 05.05.2024, Samsung A34-5G phone with android 14, CarStream and Fermata Auto is still non-functional on the car screen! Money thrown out the window.

**Comment by kaheil** [comment]
Ok looks like there's no hope here. How do I unsubscribe?

**Comment by jmckeejr** [comment]
https://www.theverge.com/2024/5/15/24157310/google-built-in-android-auto-max-peacock-rivian

Will this help?

**Comment by Nikolay-Uvarov** [comment]
Same issue in Samsung s24

**Comment by biggerald1** [comment]
It hasn't ever worked on android 14. It's been almost a year since a14 released. It's listed as not compatible w/a14. You must be on an earlier version of buy a  separate USB box.
Sent via Outlook
________________________________

**Comment by JCoyk** [comment]
How is there so many programmers that can't figure this out.  Pathetic if
you all ask me
Regards,

Jesse Coykendall, PME
Electrical Superintendent
780-237-1844

**Comment by joncue** [comment]
Coming from someone that can't put a sentence together with proper grammar, I'm pretty sure very few people care about your opinion.  I didn't see anyone ask you what you thought either, imagine that.  If it's that important to you, why don't you either:

1.  Learn how to fucking do it yourself, or
2.  Pay someone to do it for you.

Either way, quit being a prick.

**Comment by tdavisg1** [comment]
This project is clearly dead. Stop purchasing the Pro version. The website listed AAAD: Android version 13 and 14 Supported. This is false and while it installed on A14, it does not work as intended and should not be listed as supported.

**Comment by rperezan** [comment]
> This project is clearly dead. Stop purchasing the Pro version. The website listed AAAD: Android version 13 and 14 Supported. This is false and while it installed on A14, it does not work as intended and should not be listed as supported.

You should consider separating the topics. I'm here because I recently switched to a new phone, an S24, and encountered issues with CarStream. However, this problem is unrelated to AAAD, which is functioning properly and installing without any issues. I was even able to reactivate my Pro subscription within a day, thanks to the support team. While I acknowledge that the lack of video support for Android Auto might make AAAD less appealing, I'm grateful for the months of use I've gotten out of it and feel it was worth every euro spent.

**Comment by pdario** [comment]
> While I acknowledge that the lack of video support for Android Auto might make AAAD less appealing

What do you consider it appealing for, now???

**Comment by pergolafabio** [comment]
It should be abandoned indeed, last commits in code, is how to donate!!!

**Comment by rperezan** [comment]
> > While I acknowledge that the lack of video support for Android Auto might make AAAD less appealing
> 
> What do you consider it appealing for, now???

That is up to you... do not pay if you do not like it...

**Comment by pdario** [comment]
What is a single reason to pay for it at the moment? Am I missing anything?

**Comment by deusxanime** [comment]
There is a workaround I think and works for me to this day on my Pixel 9 Pro (stock, fully updated). I have an AAWireless device because my car only has wired AA built-in. If I connect through the AAWireless device rather than direct wired and have dev options enabled in both AA and in the AAWireless device/app, I can install CarStream through AAAD and it works. It absolutely destroys my battery to have AA running through wireless and trying to stream content at the same time, so I usually plug my phone in to power through a cigarette lighter/12V adapter to keep it from draining so quick (since I can't use the normal USB plug to my car as that would activate wired AA). 

The one thing that took me a while to figure out was that initially I couldn't get it working and it confused and annoyed me. I'd install CarStream via AAAD (had to break down and get AAAD Pro so I could keep fiddling uninstalling and reinstalling CS), start my car, connect to AA and it wouldn't be there in the AA app list - via either wired or wireless through AAWireless. Eventually, as I was almost ready to give up, through trial and error I figured out I had to do the CS install while AA was running via wireless. Here's what finally worked for me:

- Disconnect from everything and made sure CarStream was uninstalled completed (wipe data storage and uninstall in Settings > Apps) after many uninstalls/reinstalls from previous attempts. AAAD was still already installed on my phone though.
- Start car and connect to AA through AAWireless device
- (Make sure you already have dev mode enabled in both AA and in the AAWireless app. This may force the AAW device to reboot, but either way, if you didn't have dev mode already enabled on one or both, I'd recommend starting over and power cycling/reconnecting everything after doing that just to be sure.)
- While AA is connected and running via wireless, open AAAD and install CarStream. (This was the trick I think, whenever I tried to install CS before connecting to wireless AA, it never would show up.)
- Disconnect/reconnect/restart AA, again through AAWireless
- After that CS was available and working when connected wireless via AAW. I can use it with YT and my Plex server.
- (CS still does not show up or work of course when using standard wired AA, only while using it via the AAW device connection.)

I know this is only useful to the small number of people that have an AAW device (and this is the original even, not sure if it still works with their newer v2+ devices), but figured I'd share with the few it might be useful for.

**Comment by mdanto** [comment]
It worked for me. 

I just got a generic wireless device, one of many on Amazon. I uninstlled Carstream, made sure install unknown apps was allowed in AAAD (Settings-Apps-AAAD) and Wireless Android Auto and Unknown sources were enabled in Android Auto Developer settings (Settings-Connected devices-Android Auto-Developer settings, must enable developer settings by pressing version 10 times). I did not need to download an app to enable additional developer settings for the device. I connected to the wireless device with the phone, installed Carstream using AAAD while it was connected, then disconnected and rebooted everything and it worked. FWIW, I installed Carstream 2.0.4 by Kristakos only because 2.0.5 by Paesani2006 failed the first time. I just tried something different, but I'm pretty sure either would have worked.

**Comment by I-am-Gaming** [comment]
Carstream shows "no new messages while driving" in latest version of AAAD in Android 16

**Comment by muppetofdoom** [comment]
- Any updates on this?

**Comment by alarmymazury** [comment]
Bought the app, thats not working.... just my luck

**Comment by del-gh** [comment]
I had the same but then followed @deusxanime instructions and it's all working now. I think the main thing for me was enabling dev mode in AAWireless that did it.

**Comment by tdavisg1** [comment]
> I had the same but then followed [@deusxanime](https://github.com/deusxanime) instructions and it's all working now. I think the main thing for me was enabling dev mode in AAWireless that did it.

Can you point me to the AAW device used? Just trying to save time returning Amazon orders

**Comment by del-gh** [comment]
> > I had the same but then followed [@deusxanime](https://github.com/deusxanime) instructions and it's all working now. I think the main thing for me was enabling dev mode in AAWireless that did it.
> 
> Can you point me to the AAW device used? Just trying to save time returning Amazon orders

Sure, I got the AAWireless TWO from aawireless dot io

**Comment by mvillaser** [comment]
Same problem here on poco x7 pro Android 16, i Buy pro licence

**Comment by Dawwgmedia** [comment]
> Carstream shows "no new messages while driving" in latest version of AAAD in Android 16

I'm pretty sure it doesn't work on latest Android version. YouTube is now allowed video wise on android auto it's being rolled out this year.

**Comment by Dawwgmedia** [comment]
> Carstream shows "no new messages while driving" in latest version of AAAD in Android 16

However mate I just checked it worked via AA browser I just watched YouTube video on it lol
